src/include/sof/samples/audio/nxp_detect_test.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Cristina Feies <cristina.ilie@nxp.com>
+ * Author: Viorel Suman <viorel.suman@nxp.com>
+ */
+
+#ifndef __USER_NXP_DETECT_TEST_H__
+#define __USER_NXP_DETECT_TEST_H__
+
+#include <stdint.h>
```

Comment:
Do we want to go with `nxp_` prefix? In the past we went with `fsl_` and that didn't age well.

I would say to find a fancy and catchy name for our feature and use that instead.